FULL_HELP_DOCS.md

```diff
@@ -1043,6 +1082,42 @@ Calculate the hash of a transaction envelope from stdin
 
 
 
+## `stellar tx sign`
+
+Sign a transaction envolope appending the signature to the envelope
+
+**Usage:** `stellar tx sign [OPTIONS]`
+
+###### **Options:**
+
+* `--sign-with-key <SIGN_WITH_KEY>` — Sign with account. Alias `source`. Can be an identity (--source alice), a secret key (--source SC36…), or a seed phrase (--source "kite urban…")
+* `--sign-with-laboratory` — Sign with labratory
+* `--hd-path <HD_PATH>` — If using a seed phrase, which hierarchical deterministic path to use, e.g. `m/44'/148'/{hd_path}`. Example: `--hd-path 1`. Default: `0`
+* `--yes` — If `--sign-with-*` is used this will remove requirement of being prompted to sign transactions
+* `--rpc-url <RPC_URL>` — RPC server endpoint
```

Comment:
Yes this is true. Though it would be much work to untangle it since we make use of the network logic and resolution. Perhaps we could do this is a future, PR, but the amount of work to do this would be a bit painful. Though I completely agree it's confusing.